integration-cli/docker_api_logs_test.go

```diff
@@ -85,3 +86,94 @@ func (s *DockerSuite) TestLogsAPIContainerNotFound(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, http.StatusNotFound)
 }
+
+func extractBody(c *check.C, url string) []string {
```

Comment:
The harm is this ends up getting stuck in this package and then someone will probably come and modify it and use elsewhere even though there's other similar functions.

Can we move this into the test function?